package-dsdlsupport/src/dsdlsupport/SplunkSearch.py

```diff
@@ -116,7 +118,7 @@ def search(self, query, earliest="-15m@m", latest="now"):
         elif query_cleaned[0]=='|':
             # assume a generating search command and do nothing
             pass
-        elif query_cleaned.startswith("search "):
+        elif query_cleaned.startswith("search ") or query_cleaned.startswith("search\n"):
```

Comment:
This change is because if you do something like this (when your search gets really long):
 
```python
search = """search
something I'm looking for"""
```

It silently breaks your search (making the search "search search something I'm looking for")